### PR TITLE
Des 743

### PR DESCRIPTION
**Pull request by wyseguyonline**
Build working.

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -1,35 +1,53 @@
...
-const fontsDir          = `${__dirname}/src/styles/assets/fonts/`;
-const fonts             = fs.readdirSync(fontsDir, 'utf-8').map(font => fontsDir + font);
+const path                   = require('path');
+const webpack                = require('webpack');
+const HtmlWebpackPlugin      = require('html-webpack-plugin');
+const ExtractTextPlugin      = require('extract-text-webpack-plugin');
+const extractElementsCss     = new ExtractTextPlugin({ filename: 'css/elements.css' });
+const extractElementsNPCss   = new ExtractTextPlugin({ filename: 'css/elementsNoPlain.css' });
+const extractDemoCss         = new ExtractTextPlugin({ filename: 'css/demo.css' });
+const extractCompnentCss     = new ExtractTextPlugin({ filename: "css/elements.css" });
```
Comment (mhomolak) [inline webpack.config.js]: Typo here and where the const is used - 92 & 126.

Rest looks good and runs perfectly locally!

**Comment by JasonLantz** [comment]
Nice work!!!

**Comment by umahaea** [comment]
all approved, all regression passed: https://travis-ci.org/Pearson-Higher-Ed/ux-test-platform/builds/277888612

**Comment by umahaea** [comment]
good to merge..

**Pull request merged**